### PR TITLE
Fixes for bugs #0838914, #0838912, #0838906

### PR DESCRIPTION
**Pull request by cs-renuka-andhale**
Fixes for bugs:
1. https://mantis.fortinet.com/bug_view_page.php?bug_id=0838914
2. https://mantis.fortinet.com/bug_view_page.php?bug_id=0838912
3. https://mantis.fortinet.com/bug_view_page.php?bug_id=0838906

### CODE REVIEW

File: connectors/data.json [hunk 1]
```diff
@@ -8,64 +8,176 @@
         "operation_roles": [],
         "configurations": [
             {
-                "id": 100,
-                "config_id": "ec8bb130-6178-43c8-9deb-003690378a36",
+                "id": 3,
+                "config_id": "ef071ece-968e-4fb4-bdc7-aa84650bec83",
                 "status": 1,
-                "name": "SNOW1",
+                "name": "SNOW1_SIR",
```
Comment (cs-suyog-jinturkar) [inline connectors/data.json]: Why SNOW1?

File: connectors/data.json [hunk 2]
```diff
@@ -8,64 +8,176 @@
...
                 "default": true,
                 "config": {
-                    "password": "password",
+                    "password": "DZt7@PM*a2zp",
```
Comment (cs-suyog-jinturkar) [inline connectors/data.json:17]: Why password is mentioned here?

File: connectors/data.json [hunk 3]
```diff
@@ -8,64 +8,176 @@
...
                     "username": "admin",
-                    "server_url": "https:\/\/dev66351.service-now.com\/",
+                    "server_url": "https:\/\/dev66350.service-now.com\/",
```
Comment (cs-suyog-jinturkar) [inline connectors/data.json]: Server url is correct?

File: connectors/data.json [hunk 4]
```diff
@@ -8,64 +8,176 @@
...
                 "status": 1,
-                "name": "SNOW2",
+                "name": "SNOW_Incident",
                 "default": false,
                 "config": {
                     "password": "password",
                     "username": "admin",
-                    "server_url": "https:\/\/dev66352.service-now.com\/",
+                    "server_url": "https:\/\/dev61350.service-now.com\/",
                     "verify_ssl": false,
```
Comment (cs-suyog-jinturkar) [inline connectors/data.json:39]: Server url mentioed here too... Is it intended or by mistake

File: connectors/data.json [hunk 5]
```diff
@@ -8,64 +8,176 @@
...
+                    "scheduleId": "gAAAAABjFzlyrXX91WEvfJVe2GoCHW4BkEMwYAcVUoo3PuCC9aCJlOA6CtrWnl9lGWJyw7-QueVoYs25mnXGv9Z0Sf1SpVpdCg==",
+                    "scheduleName": "Ingestion_servicenow_SNOW1_ef071ece-968e-4fb4-bdc7-aa84650bec83",
+                    "scheduleStatus": true
+                },
+                "configuration": "ef071ece-968e-4fb4-bdc7-aa84650bec83",
+                "schedule": {
+                    "id": "gAAAAABjFzxnZKEXnRz3pFToznsfQL8dMxh850iNasEmqS3Quoi-jTtbfKlKUa_3HOMwtpXMqEW2xbKkkwcW_xrUaa-HrQ95yQ==",
+                    "crontab": {
+                        "id": 9,
+                        "minute": "*\/1",
```
Comment (cs-suyog-jinturkar) [inline connectors/data.json:72]: Every minute?

File: connectors/data.json [hunk 6]
```diff
@@ -8,64 +8,176 @@
...
                     "scheduleStatus": true
                 },
-                "configuration": "ec8bb130-6178-43c8-9deb-003690378a36",
-                "schedule": null
+                "configuration": "046e5485-a62d-4852-baab-c52a884b1dbc",
+                "schedule": {
+                    "id": "gAAAAABjFzxokEjl0deo5pxScluGKJmbfk5jGT09Itre2BigUfitsWOWTPMtQdUf53pzR-WI6GhmFsfvMvWAGI3CgrpClMfJfg==",
+                    "crontab": {
+                        "id": 9,
+                        "minute": "*\/1",
```
Comment (cs-suyog-jinturkar) [inline connectors/data.json:139]: Every minute, is it correct??

File: connectors/data.json [hunk 7]
```diff
@@ -74,176 +186,347 @@
...
                             "triggerLimit": null,
-                            "name": "ServiceNow On Incident Update",
+                            "name": "> ServiceNow > Create Incident",
                             "aliasName": null,
-                            "tag": "#servicenow #dataingestion",
-                            "description": null,
-                            "isActive": false,
-                            "debug": false,
+                            "tag": "#servicenow #dataingestion #create",
+                            "description": "Creates incident in cyops",
```
Comment (cs-suyog-jinturkar) [inline connectors/data.json]: Please change description to remove cyops n add make it record

File: connectors/data.json [hunk 8]
```diff
@@ -74,176 +186,347 @@
...
+                            "isActive": true,
+                            "debug": true,
```
Comment (cs-suyog-jinturkar) [inline connectors/data.json]: Debug false

File: connectors/data.json (hunk 2 above)
Comment (cs-renuka-andhale) [reply connectors/data.json:17]: this is dummy password.

File: connectors/data.json (hunk 3 above)
Comment (cs-renuka-andhale) [reply connectors/data.json]: dummy url is added.

File: connectors/data.json (hunk 4 above)
Comment (cs-renuka-andhale) [reply connectors/data.json:39]: dummy url is added.

File: connectors/data.json (hunk 5 above)
Comment (cs-renuka-andhale) [reply connectors/data.json:72]: by default schedule is inactive, its frequency can be changed by user as per requirement.

File: connectors/data.json (hunk 6 above)
Comment (cs-renuka-andhale) [reply connectors/data.json:139]: by default schedule is inactive, its frequency can be changed by user as per requirement.

File: connectors/data.json (hunk 7 above)
Comment (cs-renuka-andhale) [reply connectors/data.json]: updated description to Creates incident record

File: connectors/data.json (hunk 8 above)
Comment (cs-renuka-andhale) [reply connectors/data.json]: Set the debug mode to false.

File: connectors/data.json (hunk 1 above)
Comment (cs-renuka-andhale) [reply connectors/data.json]: Updated to SNOW_SIR

**Comment by cs-suyog-jinturkar** [comment]
LGTM.

**Pull request merged**